src/ui/public/timefilter/get_time.js

```diff
@@ -0,0 +1,31 @@
+import _ from 'lodash';
+import dateMath from '@kbn/datemath';
+
+export function calculateBounds(timeRange, forceNow) {
```

Comment:
I would be more explicit here for better readability and make `forceNow = false`